dags/dbt_advanced.py

```diff
@@ -6,42 +6,41 @@
 from airflow.utils.dates import timedelta
 from airflow.utils.task_group import TaskGroup
 
+DBT_PROJECT_DIR = '/usr/local/airflow/dbt'
+
 default_args = {
     'owner': 'astronomer',
     'depends_on_past': False,
     'start_date': datetime(2020, 12, 23),
     'email': ['noreply@astronomer.io'],
-    'email_on_failure': False,
-    'email_on_retry': False,
-    'retries': 1,
-    'retry_delay': timedelta(minutes=5)
+    'email_on_failure': False
```

Comment:
FYI I removed a bunch of stuff here just to keep things minimal (and make it easier for myself to debug)

Reply:
> FYI I removed a bunch of stuff here just to keep things minimal (and make it easier for myself to debug)

👍